.github/workflows/check-pl-compat.yaml

```diff
@@ -116,21 +115,6 @@ jobs:
         RT_BUILD_DIR="$(pwd)/runtime-build" \
         QIR_STDLIB_DIR="$(pwd)/qir-stdlib-build" \
         QIR_STDLIB_INCLUDES_DIR="$(pwd)/qir-stdlib-build/include" \
-        LIGHTNING_GIT_TAG_VALUE=master \
```

Comment:
As we discussed offline, let's keep both lightning tags (latest and stable) in the plugin matrix.